debian/rules

```diff
@@ -46,12 +46,6 @@ override_dh_auto_install:
 	dh_auto_install -B$(CURDIR)/build
 	find $(CURDIR)/debian/tmp -type d -empty -delete
 	rm -f $(CURDIR)/debian/tmp/usr/share/mongo-cxx-driver/LICENSE
-ifneq (,$(filter pkg.mongo-cxx-driver.mnmlstc,$(DEB_BUILD_PROFILES)))
-	# Drop MNMLSTC headers in a location where dh_install will pick them up
-	mv $(CURDIR)/debian/tmp-mnmlstc/usr/include/bsoncxx/v_noabi/bsoncxx/third_party \
-	   $(CURDIR)/debian/tmp/usr/include/bsoncxx/v_noabi/bsoncxx/
-	rm -rf $(CURDIR)/debian/tmp-mnmlstc
```

Comment:
Also remove `DESTDIR=$(CURDIR)/debian/tmp-mnmlstc` in `override_dh_auto_build` since MNMLSTIC is no longer installed during build step?

Reply:
Ooh, I missed that. I agree that it should be removed.

Reply:
Verified by [this patch](https://spruce.mongodb.com/task/cxx_driver_packaging_debian_package_build_mnmlstc_patch_05b49179910c06810189eb68468499966b03f406_6524326b2a60ed681094e201_23_10_09_17_03_40/logs?execution=0).